src/chttpd/src/chttpd_db.erl

```diff
@@ -375,8 +375,10 @@ db_req(#httpd{method='POST', path_parts=[DbName], user_ctx=Ctx}=Req, Db) ->
 db_req(#httpd{path_parts=[_DbName]}=Req, _Db) ->
     send_method_not_allowed(Req, "DELETE,GET,HEAD,POST");
 
-db_req(#httpd{method='POST',path_parts=[_,<<"_ensure_full_commit">>]}=Req, _Db) ->
+db_req(#httpd{method='POST', path_parts=[DbName, <<"_ensure_full_commit">>],
+        user_ctx=Ctx}=Req, _Db) ->
     chttpd:validate_ctype(Req, "application/json"),
+    fabric:get_security(DbName, [{user_ctx, Ctx}]),
```

Comment:
I think there is a need for a comment here to describe why the call is needed and the reason for `fabric:get_security/2` to be used for database existence check (get_security uses local shards while `fabric:get_db_info` for example uses non local shards as well)

Reply:
I'm using `get_security` here because 1) it is what old code was using 2) get security calls `fabric_util:get_db` that actually does open remote shards, but gives priority to local ones, so it allows to avoid round-trip when possible 3) it _presumably_ also confirms user to be `reader`, that what old code was saying, but I don't see it in fabric, tbh.

I can drop a comment here or I can switch to `fabric:get_db_info` if you think that'd be less subtle.